test/e2e2/flex_to_dedicated_test.go

```diff
@@ -40,7 +40,7 @@ import (
 //go:embed flex2dedicated/*
 var flex2dedicated embed.FS
 
-var _ = Describe("Flex to Dedicated Upgrade", Ordered, Label("flex-to-dedicated"), func() {
+var _ = Describe("Flex to Dedicated Upgrade", Ordered, Label("nightly-flex2dedicated", "flex-to-dedicated"), func() {
```

Comment:
Having to remember this is the biggest issue I have right now about this strategy.

The invariant is, every ´e2e2´ test must include at least one `nightly-` label. We could write a test to fail if that is not the case. I do not think it would be so complex. WDYT @helderjs ?

Reply:
Think of it as a linter that check test coverage: "are all e2e2 tests covered as nightlies?"